File: pinyougou-content-interface/src/main/java/com/lsf/pinyougou/content/service/ContentService.java
package com.lsf.pinyougou.content.service;

import java.util.List;

import com.lsf.pinyougou.pojo.TbContent;
import vo.PageResult;


/**
 * 广告服务层接口
 */
public interface ContentService {

    List<TbContent> findAll();


    PageResult findPageLimit(TbContent content, int pageNum, int pageSize);


    void add(TbContent content);


    void update(TbContent content);


    TbContent findOne(long id);


    void batchDelete(Long[] ids);


    List<TbContent> findByContentCategoryId(Long contentCategoryId);
}

File: pinyougou-search-web/src/main/webapp/js/service/contentService.js
pyg.service('contentService', function ($http) {

    // 查询指定广告分类 ID 下的所有广告
    this.findByContentCategoryId = function (categoryId) {
        return $http.get('content/findByContentCategoryId.do?categoryId=' + categoryId);
    };

});